src/controllers/roles.controller.js

```diff
@@ -1,13 +1,14 @@
-import db from '../database/models/index';
-import sendResult from '../utils/sendResult';
+import sendResult from '../utils/result.util';
 import EmailService from '../services/email.service';
+import RoleService from '../services/role.service';
+import UserService from '../services/user.service';
 
 const role = {
   changeRole: async (req, res) => {
```

Comment:
great job @urbanishimwe but let us try to make our code consistent bu using the same function type